draft-ietf-quic-qlog-h3-events.md

```diff
@@ -505,10 +505,20 @@ H3HeadersFrame = {
 ~~~
 {: #h3-headersframe-def title="H3HeadersFrame definition"}
 
+HTTP/3 can transport field names or values that are not valid; see {{Section 4.2
```

Comment:
Not valid what? Valid UTF-8? Valid semantically? valid in terms of range of bytes that are represented? etc.